python/day_7/tests/test_calem.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from crabs import fuel_cost, enhanced_fuel_cost
```

Comment:
Should you import from calem? This is importing the existing solution I think

Reply:
Yeah, nice catch ! Fixed it